docs/guide/static-deploy.md

```diff
@@ -98,24 +98,24 @@ Now the `preview` command will launch the server at `http://localhost:8080`.
          - name: Checkout
            uses: actions/checkout@v4
          - name: Set up Node
-           uses: actions/setup-node@v3
+           uses: actions/setup-node@v4
            with:
-             node-version: 18
+             node-version: 20
              cache: 'npm'
          - name: Install dependencies
-           run: npm install
+           run: npm ci
```

Comment:
I think we should revert this. `npm ci` will ignore `node_modules`, making the cache not work here.

Reply:
Maybe I'm misunderstanding this, but the documentation claims the action does not cache `node_modules`.

---

https://github.com/actions/setup-node#caching-global-packages-data
![image](https://github.com/vitejs/vite/assets/16615327/4b086c33-7f12-4aa3-bc51-8ba67cd2490a)

---

https://github.com/actions/setup-node/blob/main/docs/advanced-usage.md#caching-packages-data
![image](https://github.com/vitejs/vite/assets/16615327/c9823cf2-509c-4965-bad5-04284a87304d)

Reply:
Caching seems to work as intended with `npm ci`.

https://github.com/BoringBoredom/UEFI-Editor/actions/runs/7735912724/workflow#L39
https://github.com/BoringBoredom/UEFI-Editor/actions/runs/7735912724/job/21092359385#step:15:5
https://github.com/BoringBoredom/UEFI-Editor/actions/caches

![image](https://github.com/vitejs/vite/assets/16615327/daa6cbbe-5289-4d79-bccc-abc013403504)
![image](https://github.com/vitejs/vite/assets/16615327/b5e833d7-7ee9-434a-867f-b33536941f79)
![image](https://github.com/vitejs/vite/assets/16615327/93313597-ba0b-490b-8211-67031c825192)

Reply:
I think it's fine to use `npm ci`.